backend/backend/core/tools.py

```diff
@@ -41,35 +48,35 @@ def __init__(self):
         super().__init__(
             name="python_repl",
             description="""
-            A Python repl tool to execute python scripts.
-            Use this to perform computations or solve problems through programming.
-            Input should be a valid python command or script.
-            Beware of syntax errors!
-            Only use built-in python functions and libraries.
-            If you want to see the output of a value,
-            you should print it out with `print(...)`.
+A Python repl tool to execute python scripts.
+Use this to perform computations or solve problems through programming.
+Input should be a valid python command or script.
+Beware of syntax errors!
+Only use built-in python functions and libraries.
+If you want to see the output of a value,
+you should print it out with `print(...)`.
             """,
             func=PythonREPL().run,
         )
 
 
-class GitHubCommentInput(BaseModel):
-    repo: str = Field(..., description="The repository name in 'owner/repo' format")
-    pr_number: int = Field(..., description="The pull request number")
-    comment: str = Field(..., description="The comment to add")
+class GitHubCommentTool(BaseTool):
+    """
+    A tool to add comments to GitHub pull requests.
+    """
 
+    class Comment(BaseModel):
+        repo: str = Field(..., description="The repository name in 'owner/repo' format")
+        pr_number: int = Field(..., description="The pull request number")
+        comment: str = Field(..., description="The comment to add")
 
-class GitHubCommentTool(BaseTool):
-    name: str = "github_comment"
-    description: str = "A tool to add comments to GitHub pull requests."
-    args_schema: Type[BaseModel] = GitHubCommentInput
+    name: str = "github_pullrequest_comment"
+    description: str = "A tool to generic comments to GitHub pull requests."
```

Comment:
The description for `github_pullrequest_comment` should be corrected from 'A tool to generic comments' to 'A tool to add comments'.